test/cli/test_catalog.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from unittest.mock import patch, MagicMock
+from click.testing import CliRunner
+from ersilia.cli.commands.catalog import catalog_cmd
+
+@pytest.fixture
+def runner():
+    return CliRunner()
+
+@patch("ersilia.hub.content.catalog.ModelCatalog.airtable", return_value=None)
+@patch("ersilia.hub.content.catalog.ModelCatalog.local")
+@patch("ersilia.hub.content.catalog.ModelCatalog.hub")
+@patch("ersilia.hub.content.card.ModelCard.get", return_value={"model": "metadata"})
```

Comment:
This information is present on the disk as well, especially for the models available locally.